site/documentation/static/api/device-registry-v1.yaml

```diff
@@ -629,16 +650,28 @@ components:
                  "hash-function":
                     type: string
                     example: sha-512
+                    description: The name of the hash function used to create the password hash (defined in 'pwd-hash' property).
+                                 If password is defined using 'pwd-plain' property, this value will be ignored by the device registry.
+                                 This property should be empty when returning passwords from the device registry.
```

Comment:
I would prefer pointing out the two choices here, like:

This property may be empty on read results, if the device registry conceals authentication secrets. In this case the `id` field must be set instead.

Reply:
Yes, I forgot to update it. Thanks!